src/dash-adapter.js

```diff
@@ -365,10 +372,42 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     }
     this._maybeSetFilters();
     this._maybeSetDrmConfig();
+    this._maybeBreakStalls();
     this._shaka.configure(this._config.shakaConfig);
     this._addBindings();
   }
 
+  _clearStallInterval(): void {
+    if (this._stallInterval) {
+      clearInterval(this._stallInterval);
+      this._stallInterval = null;
+    }
+  }
+
+  _stallSmartTVHandler(): void {
+    this._clearStallInterval();
+    let lastCurrentTime = this._videoElement.currentTime;
+    this._stallInterval = setInterval(() => {
+      if (lastCurrentTime === this._videoElement.currentTime) {
+        this._videoElement.currentTime = parseFloat(this._videoElement.currentTime.toFixed(1)) + 0.1;
+      } else {
+        this._clearStallInterval();
+      }
+    }, 2 * 1000);
```

Comment:
move to const, maybe expose as config?

Reply:
Didn't want to do it too robust cause I believe we need to handle it on Shaka, WDYT?

Reply:
To create 3 configs for hack we're adding for edge case that we won't need anymore after Shaka will handle it, it sounds too robust for me, don't you think?

Reply:
agree, let's start lean, but please move to consts and we will hope Shaka will approve this :-)